IPython/core/display.py

```diff
@@ -622,7 +645,7 @@ def _pngxy(data):
 def _jpegxy(data):
     """read the (width, height) from a JPEG header"""
     # adapted from http://www.64lines.com/jpeg-width-height
-    
+
```

Comment:
Some of the IPython devs won't be happy about the whitespace changes here (not me), you may want to strip these out.